packages/playground-examples/copy/ja/TypeScript/Primitives/Any.ts

```diff
@@ -0,0 +1,51 @@
+// AnyはTypeScriptのエスケープ句です。 
+// any型を利用することで、一区切りのコードを
+// JavaScriptの様に動的に扱ったり、
+// 型システムの制限を回避することができます。
+
+// any型を利用する良いケースはJSON parsingです
+
+const myObject = JSON.parse("{}");
+
+// TypeScriptにおけるAny宣言は、あなたがその値について詳しく知っていて
+// 安全なものなので信じてください、という宣言です。
+// それが厳密に正しくない場合でも。例えば、次のコードはクラッシュします。
+
+myObject.x.y.z;
+
+// any型を利用することで、型安全のトレードオフと引き換えに、
+// よりオリジナルに近いJavaScriptコードを書くことができます。
+
+// any型は、どんな型でも（neverを除いた）割り当て可能であり、
```

Comment:
[imo] 括弧はあってもなくても成立する文章にしたいです
```suggestion
// any型は、（neverを除いて）どんな型でも割り当て可能であり、
```

Reply:
コメントありがとうございます！該当箇所を修正しました